server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java

```diff
@@ -204,12 +208,183 @@ public void runAddAnyMembershipTypeTransaction_owner() throws Exception {
+      StorageHandler.runDeleteMembershipTransaction(dbClient, "person", "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_clubDoesNotExistInDb() throws Exception {
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.getPerson(dbClient, "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_none() throws Exception {
+    StorageHandlerTestHelper.insertClub("club", /* owner_id= */"owner");
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.getListOfMembers(dbClient, "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_oneOwner() throws Exception {
+    List<Person> expected = new ArrayList<Person>();
+    expected.add(StorageHandlerTestHelper.createTestPersonObject("owner", /* pronouns= */true));
+
+    StorageHandlerTestHelper.insertPerson("owner");
+    StorageHandlerTestHelper.insertClub("club", /* owner_id= */"owner");
+    StorageHandlerTestHelper.insertMembership("owner", "club", MembershipConstants.OWNER);
+    List<Person> actual =  new ArrayList<Person>(StorageHandler.getListOfMembers(dbClient, "club"));
+
+    assertEquals(1, actual.size());
+    assertEquals(expected.get(0).getNickname(), actual.get(0).getNickname());
+    assertEquals(expected.get(0).getEmail(), actual.get(0).getEmail());
+    assertEquals(expected.get(0).getUserId(), actual.get(0).getUserId());
+    assertEquals(expected.get(0).getPronouns().get(), actual.get(0).getPronouns().get());
+  }
+
+  @Test
+  public void getListOfMembers_ownerAndMembers() throws Exception {
+    List<Person> expected = new ArrayList<Person>();
+    expected.add(StorageHandlerTestHelper.createTestPersonObject("member", /* pronouns= */true));
+    expected.add(StorageHandlerTestHelper.createTestPersonObject("owner", /* pronouns= */true));
+
+    StorageHandlerTestHelper.insertPerson("owner");
+    StorageHandlerTestHelper.insertClub("club", /* owner_id= */"owner");
+    StorageHandlerTestHelper.insertMembership("owner", "club", MembershipConstants.OWNER);
+
+    StorageHandlerTestHelper.insertPerson("member");
+    StorageHandlerTestHelper.insertMembership("member", "club", MembershipConstants.MEMBER);
+    List<Person> actual =  new ArrayList<Person>(StorageHandler.getListOfMembers(dbClient, "club"));
+
+    assertEquals(2, actual.size());
+    assertEquals(expected.size(), actual.size());
+
+    for (int i = 0; i < actual.size(); i++) {
+      assertEquals(expected.get(i).getNickname(), actual.get(i).getNickname());
+      assertEquals(expected.get(i).getEmail(), actual.get(i).getEmail());
+      assertEquals(expected.get(i).getUserId(), actual.get(i).getUserId());
+      assertEquals(expected.get(i).getPronouns().get(), actual.get(i).getPronouns().get());
+    }
+  }
+
+  @Test
+  public void getListOfClubs_memberOfAll() throws Exception {
+    List<Club> expected = new ArrayList<Club>();
+    expected.add(StorageHandlerTestHelper.createTestClubObject("clubs1", /* contentWarning= */true));
```

Comment:
nit: for this variable you're using camelCase, but for owner_id you're using snake case. consider using one throughout the code for stylistic consistency.